### ISSUE TITLE
How to change the max epoch when training?

### ISSUE THREAD

**Issue by zhouyingji** [junyanz/pytorch-CycleGAN-and-pix2pix#1049]
The default epoch is 200, but how to change it? I can't find which parameter to adjust it. 
IF I want to training 400 epoch?
Thank you

**Comment by smitheric95** [comment]
From the `train.py --help` output: 

```
  --n_epochs N_EPOCHS   number of epochs with the initial learning rate
                        (default: 100)
  --n_epochs_decay N_EPOCHS_DECAY
                        number of epochs to linearly decay learning rate to
                        zero (default: 100)
```

So in your case, `python train.py --n_epochs 300 --n_epochs_decay 100`

**Comment by zhouyingji** [comment]
> From the train.py --help output:
>   --n_epochs N_EPOCHS   number of epochs with the initial learning rate
>                         (default: 100)
>   --n_epochs_decay N_EPOCHS_DECAY
>                         number of epochs to linearly decay learning rate to
>                         zero (default: 100)
> 
> So in your case, python train.py --n_epochs 300 --n_epochs_decay 100

Get it! Thank you!
